.github/update-documentation.yml

```diff
@@ -0,0 +1,26 @@
+name: Update Documentation
```

Comment:
Not sure Update Documentation makes sense as the name. Maybe something like `Create Release`